packages/postgrest/lib/src/postgrest_builder.dart

```diff
@@ -212,7 +212,9 @@ class PostgrestBuilder<T, S> implements Future<T> {
       int? count;
 
       if (response.request!.method != METHOD_HEAD) {
-        if (response.request!.headers['Accept'] == 'text/csv') {
+        if (response.bodyBytes.isEmpty) {
```

Comment:
@Vinzent03, this looks like it will address the exception. Exceptions do interrupt the debugging flow when there is actually nothing exceptional happening.

Thanks